build.gradle

```diff
@@ -101,6 +101,8 @@ allprojects {
     // Default to the apache license
     project.ext.licenseName = 'The Apache Software License, Version 2.0'
     project.ext.licenseUrl = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+    // Get OpenSearch version catalog
+    project.getExtensions().getExtraProperties().set("versions", VersionProperties.getVersions())
```

Comment:
nit: not 100% sure this is a must as AD uses versions.<package name> toget versions from opensearch core but doesn't have this line https://github.com/opensearch-project/anomaly-detection/blob/main/build.gradle#L215

Reply:
That said, I just copied from a working config and it seems removing that line it still builds.